src/dropdown/dropdown.spec.ts

```diff
@@ -274,6 +274,28 @@ describe('ngb-dropdown-toggle', () => {
          expect(dropdownEl).toHaveCssClass('open');
        });
      })));
+  it('should have css class dropdown on div and dropdown-toggle on button',
```

Comment:
This test should be moved to the ` describe('ngb-dropdown'` section and made much simpler (for example we don't need to have `ngbDropdown` inside at all)